app/src/main/java/io/pumpkinz/pumpkinreader/NewsListFragment.java

```diff
@@ -128,57 +130,10 @@ public void onError(Throwable e) {
             @Override
             public void onNext(List<News> items) {
                 Log.d("stories", String.valueOf(items.size()));
-                Log.d("stories content", items.toString());
+                //Log.d("stories content", items.toString());
```

Comment:
Master branch's code is production ready, hence we should not leave any dirty code like unused comment there.